src/SignalR/perf/benchmarkapps/Directory.Build.props

```diff
@@ -1,2 +0,0 @@
-<Project>
```

Comment:
Move these to the `BenchmarkServer` folder. We want to keep them for that project.

Reply:
That project does not build without them, as it uses `Reference`...

Reply:
That project isn't really meant to be used by itself. It's part of our benchmarks https://github.com/aspnet/Benchmarks

Reply:
Odd - I'm using it to point Crankier at.  Also see #9264.

Reply:
Oh... don't touch that project please

Reply:
done